src/components/experimental/Divider/docs/Divider.stories.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { StoryObj, Meta } from '@storybook/react';
+import { Divider } from '../Divider';
+import { Text } from '../../Text/Text';
+
+const meta: Meta = {
+    title: 'Experimental/Components/Divider',
+    component: Divider,
+    parameters: {
+        layout: 'centered'
+    }
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Divider>;
+
+export const Default: Story = {
+    decorators: [
+        Story => (
+            <div style={{ width: '6rem', height: '6rem' }}>
+                <Text>One</Text>
+                <Story />
+                <Text>Two</Text>
+            </div>
+        )
+    ],
+    args: {
+        vertical: false
+    }
+};
+
+export const Inset: Story = {
+    decorators: [
+        Story => (
+            <div style={{ width: '6rem', height: '6rem' }}>
+                <Text>One</Text>
+                <Story />
+                <Text>Two</Text>
+            </div>
+        )
+    ],
+    args: {
+        vertical: false,
+        variant: 'inset'
+    }
+};
+
+export const MiddleInset: Story = {
+    decorators: [
+        Story => (
+            <div style={{ width: '6rem', height: '6rem' }}>
+                <Text>One</Text>
+                <Story />
+                <Text>Two</Text>
+            </div>
+        )
+    ],
+    args: {
+        vertical: false,
+        variant: 'middle-inset'
+    }
+};
+
+export const VerticalDivider: Story = {
+    args: {
+        vertical: true
+    },
+    decorators: [
+        Story => (
+            <div
+                style={{
+                    display: 'flex',
+                    height: '6rem',
+                    width: '6rem',
+                    alignItems: 'center'
+                }}
+            >
+                <Text>One</Text>
+                <Story />
+                <Text>Two</Text>
+            </div>
+        )
+    ]
+};
+
+export const VerticalInset: Story = {
```

Comment:
The weird thing is I couldn't find any vertical dividers with these variants in the wild :( I'd say let's merge it and see how it evolves?

Reply:
You know my feelings on the topic 😹 I honestly think that the variant in its entirety is pointless here (considering we can use space and layout props), so I don't mind either way.. 

but in this specific case, it's probably harder to enforce the idea that the "variant" only makes sense for horizontal dividers and that it doesn't do anything for vertical dividers (unless we treat them as an error perhaps), so I would suggest leaving this in, if we never turn out to use it, we can probably remove it later when we do a learned refactor